izumi-reflect/izumi-reflect/src/main/scala-2/izumi/reflect/TagMacro.scala

```diff
@@ -453,10 +453,43 @@ class TagMacro(val c: blackbox.Context) {
     tpeSymbol
   }
 
+  protected[this] def mkTypeParameter(owner: Symbol, tpe: Type): Symbol = {
```

Comment:
`protected[this] def mkTypeParameter(owner: Symbol, kind: Kind)` is now never used. The only condition under which it's called is never true.